src/main/java/com/pmm/budgetcalculator/expensing/application/ExpenseController.java

```diff
@@ -0,0 +1,37 @@
+package com.pmm.budgetcalculator.expensing.application;
+
+import com.pmm.budgetcalculator.expensing.domain.expense.Expense;
+import com.pmm.budgetcalculator.expensing.domain.expense.ExpenseService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Objects;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/expenses")
+public class ExpenseController {
+    private final ExpenseService expenseService;
+
+//    @Autowired
+//    ExpenseService expenseService;
+//
+//    @Autowired
+//    ExpenseRepository expenseRepository;
+
+    @GetMapping
+    public ResponseEntity<List<Expense>> getAllExpenses(
+            @RequestParam(value = "branchId", required = false) Long expenseId) {
```

Comment:
Few things here:
1. branchId is not very good name for expenseId parameter
2. Instead of adding request parameter here you can use separate import for getBy id with  path variable ex.
    @GetMapping("/{id}")
    public ReponseEntity<Expense> getOneExpense(@PathVariable("id") Long id) {//body}
3. RequestParameter can be used if you want to search ofr given expense by name or description for example.